test/multi_node_test_2.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+
+import ray
+import ray.services as services
+from ray.test.cluster_utils import Cluster
+
+logger = logging.getLogger(__name__)
+
+
+def test_cluster():
+    """Basic test for adding and removing nodes in cluster."""
+    g = Cluster(initialize_head=False)
```

Comment:
Let's modify these tests to start all nodes with 1 CPU and a small amount of memory (e.g., 100MB). I think we want that to be the normal thing to do because we may be creating many many nodes on a single machine and don't want to use up all the resources.